code/_helpers/global_access.dm

```diff
@@ -19,86 +19,10 @@
 			return global.DEPT_FREQS;
 		if("Debug2")
 			return global.Debug2;
-		if("Failsafe")
-			return global.Failsafe;
-		if("GLOB")
```

Comment:
isn't this constantly used everywhere?  so are the subsystems

Reply:
We don't need because GLOBAL system already do it thing.